src/EnergyPlus/InputProcessor.hh

```diff
@@ -23,6 +24,7 @@
 // EnergyPlus Headers
 #include <EnergyPlus.hh>
 #include <DataGlobals.hh>
+#include <GlobalNames.hh>
```

Comment:
Project doesn't build without it. There are files where we use InputProcessor:: we also use data structures form GlobalNames. There are more then one file. So if we want to remove this library from InputProcessor we will have to go through all those files that use Data Structs from GlobalNames and add that library.